src/components/views/Achievements.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { Button } from "../ui/Button";
+import "styles/views/Login.scss";
+import "styles/views/Profile.scss";
+import BaseContainer from "../ui/BaseContainer";
+import { api, useError } from "../../helpers/api";
+import User from "../../models/User";
+import PropTypes from "prop-types";
+import Achievement from "../../models/Achievement";
+import "styles/views/Achievements.scss";
+import IMAGES from "../../assets/images/index1.js";
+import ICONS from "../../assets/icons/index.js";
+
+const AchievementItem = ({ achievement }) => (
+    <div className="achievement">
+        <div className="achievement icon">
+            <img
+                src={
+                    IMAGES[achievement.profilePicture] ||
+                    ICONS[achievement.profilePicture]
+                }
+                alt="achievement icon"
+            />
+        </div>
+        <div className="achievement details">
+            <div className="achievement title">{achievement.title}</div>
+            <div className="achievement description">
+                {achievement.description}
+            </div>
+        </div>
+    </div>
+);
+
+AchievementItem.propTypes = {
+    achievement: PropTypes.object,
+};
+
+const Achievements = () => {
+    const navigate = useNavigate();
+    const [achievements, setAchievements] = useState([]);
+    const [userAchievements, setUserAchievements] = useState([]);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const userId = localStorage.getItem("userId");
+                let response = await api.get("/users/" + userId);
+                const user = new User(response.data);
+                let responseAchievement = await api.get("/users/achievments");
+                setAchievements(responseAchievement.data);
+                setUserAchievements(user.achievements);
+            } catch (error) {
+                console.log(error);
```

Comment:
maybe remove the console.log here. If it should be logged, maybe do that in useError?